docs/DeveloperGuide.adoc

```diff
@@ -169,7 +169,9 @@ image::LogicClassDiagram.png[width="800"]
 _Figure 2.3.1 : Structure of the Logic Component_
 
 image::LogicCommandClassDiagram.png[width="800"]
-_Figure 2.3.2 : Structure of Commands in the Logic Component. This diagram shows finer details concerning `XYZCommand` and `Command` in Figure 2.3.1_
+_Figure 2.3.2 : Structure of Commands in the Logic Component_
+
+Figure 2.3.1 shows the structure of the `Logic` component, while Figure 2.3.2 shows finer  details concerning `XYZCommand` and `Command` in Figure 2.3.1.
```

Comment:
Should we break this up and chunk it under respective diagrams ? Or this is fine?